everviz/plugins/lineplot/lineplot.py

```diff
@@ -0,0 +1,102 @@
+import os
+import pkg_resources
+
+import dash_html_components as html
+import dash_core_components as dcc
+
+import plotly.graph_objs as go
+import plotly.express as px
+from pathlib import Path
+from uuid import uuid4
+from dash.dependencies import Output, Input
+from webviz_config import WebvizPluginABC
+from webviz_config.webviz_assets import WEBVIZ_ASSETS
+from everviz.data.load_csv.get_data import get_data
+
+
+class Lineplot(WebvizPluginABC):
+    def __init__(self, app, data_path, title="Lineplot"):
+        super().__init__()
+        self.title = title
+        self.graph_id = f"graph-{uuid4()}"
+        self.filter_box_id = f"dropdown-{uuid4()}"
+        self.radio_id = f"dropdown-{uuid4()}"
+
+        self.data_path = data_path
+        self.set_callbacks(app)
+
+        ASSETS_DIR = Path(
+            pkg_resources.resource_filename("everviz", os.path.join("assets"))
```

Comment:
why the `os.path.join()` call?

Reply:
Good catch, will remove

> Looks good to me, some things may need discussion, but I will approve since we can always fix things later. For instance, is the name LinePlot maybe not too generic?
> 
> We end up with small differences in our plugins I think, for instance you use P50, I use the mean. There are arguments for both, we should find out what the users want.
> 
> So maybe we should empty our issues list, merge everything and then get together to align.

Yes, did notice a lot of synergies between the two, and yes, we should be able to come up with something. I´ll address your other comments today, but hold off on merging until we´ve had a discussion tomorrow if you have time?